update-bot/config/templates.json

```diff
@@ -36,8 +36,8 @@
       },
       "Denkmalliste Sachsen-Anhalt Tabellenzeile": {
           "id": "ID",
-          "id_check": "\\d{8}",
-          "id_check_description": "achtstellige Nummer"
+          "id_check": "\\d{3}\\s*\\d{5}",
```

Comment:
Currently this regex also matches the format `nnnnnnnn` (no space due to the use of `*` instead of `+`). Ignore this comment if this was intended :)